style.css

```diff
@@ -0,0 +1,69 @@
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #2b57a8;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text {
+  width: 327px;
+  height: 150px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.head-section {
+  width: 327px;
+  height: 146px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
```

Comment:
- Kinldy make sure your section has the same as the figma design like figma design as recommended be the [project  requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates) 
  - Your design:
![main](https://user-images.githubusercontent.com/97834160/205600621-a8f3286a-69ad-48db-904a-3bbee5efcad2.png)


  - Figma:
![mainFig](https://user-images.githubusercontent.com/97834160/205600685-54a73f61-ad8e-4c1e-8d77-dfcb56d23e9b.png)